### PR TITLE
Conseil-166 Start updating and clean up the current README

### PR DESCRIPTION
**Pull request by ivanopagano**
Partly address #166 

- [ ] Update, correct and clean up the Readme information
- [ ] Write a User Manual for Conseil v2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ Query API for the Tezos blockchain
 ## Running Conseil
 
 Conseil has two entry points:
-- `src/main/tech/cryptonomic/Conseil/Conseil.scala` runs a server exposing a RESTful API.
-- `src/main/tech/cryptonomic/Conseil/Lorre.scala` is a background process that runs on loop and writes Tezos blockchain data to database.
+- `src/main/tech/cryptonomic/Conseil/Conseil.scala` runs a server exposing a RESTful API from locally stored blockchain data.
```
Comment (vishakh) [inline README.md]: What does "locally sourced" mean?

File: README.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ Query API for the Tezos blockchain
...
+- `src/main/tech/cryptonomic/Conseil/Lorre.scala` is a background process that loops and synchronize (Tezos) blockchain data to database.
```
Comment (vishakh) [inline README.md]: synchronize => synchronizes

File: README.md [hunk 3]
```diff
@@ -17,77 +17,108 @@ The Conseil server should be run behind a proxy such as Nginx with TLS enabled t
...
 
 Deployment
-- JRE (>7.x)
+- JRE (> 8.x)
 - A database supported by Typesafe Slick, e.g. Postgres
 
-### Building, packing and deploying Conseil
-
-#### Compile
+## Building, packing and deploying Conseil
```
Comment (vishakh) [inline README.md]: Can we add a 'Package' header to this section, i.e. `#### Package`?

File: README.md (hunk 1 above)
Comment (ivanopagano) [reply README.md]: I meant to say that it uses a separate db over reading the data from the chain itself...
I guess I might rephrase to be more correct...

**Pull request merged**